tensorflow/tf_version_script.lds

```diff
@@ -2,6 +2,7 @@ tensorflow {
   global:
     *tensorflow*;
     *perftools*gputools*;
+    TF_*;
```

Comment:
Actually, I don't know whether it's case sensitive. I'll look around for documentation.